config/containers/logging/dual-logging.md

```diff
@@ -0,0 +1,116 @@
+---
+description: Learn how to read container logs locally when using a third party logging solution.
```

Comment:
I think this entire file belongs to the `docs-private` repo, since reading the logs for remote drivers is a Docker EE engine feature. @thaJeztah : Can you confirm that we didnt port this feature to CE and moby? Thanks.

Reply:
@paigehargrave : This file should not be part of the public moby docs. It should be part of the EE engine docs. You should create another PR in the EE repo with this file and link the rest of the content to that link,

Reply:
@anusha-ragunathan The EE and CE engine information both live in docs.docker.com - they are not separate at this point. I've included a note about support for CE vs. EE, please let me know if more is needed. Thank you!

Reply:
Where is the note about support for CE vs EE? I think the beginning of this page should make it obvious that you need a Docker EE Engine for this feature. Maybe have a prerequisite section?

Reply:
It's in a note in the overview section.

Reply:
I changed the note to 'Prerequisites'.